### PR TITLE
Downgrade CLI version because of https://github.com/NuGet/NuGet.Clien…

### PR DESCRIPTION
**Pull request by BrennanConroy**
…t/pull/289

@pranavkm @dougbu

### CODE REVIEW

**Comment by davidfowl** [comment]
This is such a big downgrade...

**Comment by BrennanConroy** [comment]
Would you like to find a version in between that doesn't have any bugs and works for all our repos?

**Comment by davidfowl** [comment]
I rather force the fix to go in than regress a ton of CLI functionality.

**Comment by dougbu** [comment]
Is a fix ready? What exactly will we lose, given Travis is on the floor with this CLI version or any other?

File: KoreBuild-dotnet/build/KoreBuild.cmd [hunk 1]
```diff
@@ -41,6 +41,8 @@ CALL %~dp0dotnet-install.cmd -Channel %KOREBUILD_DOTNET_CHANNEL% -Version %KOREB
 
 ECHO Adding %DOTNET_LOCAL_INSTALL_FOLDER_BIN% to PATH
 SET PATH=%DOTNET_LOCAL_INSTALL_FOLDER_BIN%;%PATH%
+ECHO Setting DOTNET_HOME to %DOTNET_LOCAL_INSTALL_FOLDER%
+SET DOTNET_HOME=%DOTNET_LOCAL_INSTALL_FOLDER%
```
Comment (dougbu) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: Thought we only just removed this setting (perhaps because it was never read)?

**Comment by davidfowl** [comment]
Travis is already on the floor, what I don't want is to go back that many versions without understanding the impact. What fixes are you regressing by this downgrade?

**Comment by kichalla** [comment]
I suppose this is a fix for the `end of central directory not found`? Do we need to downgrade though? Yesterday @pranavkm found the issue to be with MyGet actually and if you point your feeds to our build shares this problem went way. This is what fixed MusicStoreTest yesterday.

**Pull request closed without merging**